Casks/o/openframeworks.rb

```diff
@@ -14,7 +14,7 @@
   # than using the `GithubReleases` strategy to check multiple recent releases.
   livecheck do
     url "https://openframeworks.cc/download/"
-    regex(/href=.*?of[._-]v?(\d+(?:\.\d+)+)[._-]osx[._-]release\.zip/i)
+    regex(/href=.*?of[._-]v?(\d+(?:\.\d+)+)[._-]osx[._-]release\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?of[._-]v?(\d+(?:\.\d+)+)[._-]osx[._-]release\.(?:t|zip)/i)
```

When upstream has used more than one file type over time, I sometimes allow the regex to match all of the applicable extensions unless it causes an issue for matching (or it's clear that only one file type will be used going forward). In this case, the `osx_release` suffix won't appear in a file for another OS, so we shouldn't have to worry much about unexpected matches from this approach.

Reply:
Done!  

What's your thoughts on standardizing on a pattern for the file suffix so we can avoid some of these easier ones?  Such as `(?:dmg|pkg|zip)`?

I think I'd rather the downloads fail than the livecheck fail (and even worse when it matches an old pattern and we don't even know there was an update)

Reply:
I've used `(?:dmg|pkg)` when a project has switched from one type to the other and I've thought about standardizing it before (i.e., it would allow the regex to continue matching if upstream switches between the formats) but I wasn't sure if it would cause any unexpected problems when rolled out more broadly or if it would be worth the effort (and added verbosity). If it's unlikely to cause problems, then replacing `\.dmg`/`\.pkg` in regexes with `\.(?:dmg|pkg)` may be fine. That said, there are currently only ~9 instances of `\.(?:dmg|pkg)`, so it's also probably fine to continue using it reactively (rather than proactively modifying related regexes).

However, I don't think we can safely standardize on a pattern that includes `|zip`, as there's no guarantee that a zip file will be for macOS. We could technically include it when projects include an OS suffix in the filename (e.g., example-1.2.3-mac.dmg) but some projects use a simple filename where the extension is the only way to distinguish between platforms (e.g., Example.zip for Windows or Linux, Example.dmg for macOS, etc.). Including `|zip` could lead to false positives (e.g., livecheck surfacing a new version that only applies to other OSs) and I generally try to avoid those because they can't be readily identified just from looking at livecheck output (i.e., they only become apparent when you try the download URL, check GitHub release assets, etc.).

This may just be me but I typically prefer for livecheck to fail rather than produce a false positive, as livecheck failures are very visible and don't lead to unnecessary work. A false positive would cause the autobump workflow to try to fetch a file that doesn't exist and this would continue to happen on every autobump run until someone notices and addresses it. Either way, we wouldn't update to a new version until someone looks into the situation and modifies the cask, so livecheck failures shouldn't be an issue in that case.